changelog/issue-4595.md

```diff
@@ -0,0 +1,16 @@
+audience: users
+level: minor
+reference: issue 4595
+---
+Generic Worker can now be run in headless mode, meaning tasks do not have a
+dedicated graphical user environment. To do this, the Generic Worker config
+setting `headlessTasks` should be set to true. This can only be enabled or
```

Comment:
There's part of me that's thinking this config option should just be `headless`. But I'm not fully convinced one way or the other, just wanted to toss it out there and hear why you thought to use `headlessTasks`?

Reply:
My thinking was the config settings relates to the execution of the tasks, rather than the execution of the worker (e.g. other config settings manage how the worker behaves, rather than how the tasks behave).